sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -2522,7 +2522,7 @@ CurlConnection::CurlConnection(
   if (performResult != CURLE_OK)
   {
 #if defined(AZ_PLATFORM_LINUX)
-    if (performResult == CURLE_SSL_PEER_CERTIFICATE)
+    if (performResult == CURLE_PEER_FAILED_VERIFICATION)
```

Comment:
shouldn't we check for both ? since it currently builds i would suspect the value still exists int he curl code we might accidentally ignore it .

Reply:
No, we shouldn't check for both.
The issue has the necessary details here. This old symbol is mapped to the new one in curl, so it is exactly the same (with the same underlying value). I want to avoid using an undocumented symbol, that is explicitly deprecated.

See this link:
https://github.com/curl/curl/blob/565d28dc8e910386d9d215e0440291bb8c4238f8/include/curl/curl.h#L661-L663